### PR TITLE
Fix #1074 Migrate article: How much downtime do maintenance operation…

### LINKED ISSUE #1074: Migrate article: How much downtime do maintenance operations cause?
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/495336

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by sluhur**
…s cause? Fix #1074

# What changed, and why it matters
Fix #1074

### CODE REVIEW

**Comment by sluhur** [comment]
@ftisiot This is the new PR

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur, should this be a `concept`? 

I left a couple of comments

File: docs/platform/reference/maintenance-downtime.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Maintenance downtime window
+===========================
+
+Maintenance window is a time window during which we'll switch to new upgraded virtual machines and point the connection URI DNS name at the new server.
```
Comment (ftisiot) [inline docs/platform/reference/maintenance-downtime.rst:4]: When is this happening? Can we give some examples?  we do it to apply patches?

File: docs/platform/reference/maintenance-downtime.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Aiven service upgrades are performed in rolling forward style, which means that new service nodes are first created alongside with the older nodes one at a time, after which the old nodes are retired.
```
Comment (ftisiot) [inline docs/platform/reference/maintenance-downtime.rst:6]: You can see an example for apache Kafka here https://docs.aiven.io/docs/products/kafka/concepts/upgrade-procedure.html

File: docs/platform/reference/maintenance-downtime.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+In case of **MySQL®**, **PostgreSQL®** and **Redis®*** the maintenance window usually lasts around 5-10 seconds. The downtime comes from old master stopping itself in a controlled manner and new master executing promotion sequence after this. Once the promotion is complete the old master node starts forwarding requests to the new master node so the service is accessible before DNS updates are propagated, though clients that end up reconnecting to the old master node will see additional disconnection once the old master is permanently retired.
```
Comment (ftisiot) [inline docs/platform/reference/maintenance-downtime.rst:8]: Are we sure about the timeline here? Or should we be more conservative saying "some seconds" without any fixed limit?

File: docs/platform/reference/maintenance-downtime.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+In case of **Kafka®** and **OpenSearch®** the service DNS address resolves to all the available service nodes. During an upgrade the DNS address changes to reflect the added and removed nodes. For example, a three node plan will have a minimum of three nodes available at all times during the whole upgrade operation.
```
Comment (ftisiot) [inline docs/platform/reference/maintenance-downtime.rst:10]: Kafka -> Apache Kafka, and again, they can see the example procedure here  https://docs.aiven.io/docs/products/kafka/concepts/upgrade-procedure.html

File: docs/platform/reference/maintenance-downtime.rst [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+Note: while the DNS name remains the same, the IP address it points to, will change during a maintenance break.
```
Comment (ftisiot) [inline docs/platform/reference/maintenance-downtime.rst:12]: Can we move this to be a proper devportal note?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur, should this be a `concept`? 

I left a couple of comments

File: docs/platform/reference/maintenance-downtime.rst (hunk 4 above)
Comment (sluhur) [reply docs/platform/reference/maintenance-downtime.rst:10]: I've put the example above.

**Comment by sluhur** [comment]
> Hi @sluhur, should this be a `concept`?
> 
> I left a couple of comments

Move this to concept

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**